### DISCUSSION TITLE
Input stream does not have enough bytes to parse declared contents of the 00 marker

### DISCUSSION THREAD

**Discussion by h3yun** [SixLabors/ImageSharp#2303] [Q&A]
I find the problem fixed in #2147 ,and the latest package version is 2.1.3. But it seems is not contained

**Comment by br3aker** [comment]
Looking at the release docs seems it's yet to be included in one of the upcoming release(s). If possible can you please manually build ImageSharp from master and check whether mentioned PR fixed image decoding for you?

**Reply by JimBobSquarePants** [reply]
2.1.3 was a hotfix build that does not contain #2147 

The fix will be in V3. They can grab a pre-built binary [here](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp).

**Reply by br3aker** [reply]
Completely forgot about pre-releases, thanks!

**Reply by h3yun** [reply]
thank you!
